proj/src/main/webapp/test/signup.test.js

```diff
@@ -99,7 +99,7 @@ describe('Test password validations', function() {
         assert.isTrue(signup.meetRequirements('testing1!'))
     });
     it('password too short', async function() {
-        assert.isFalse(signup.meetRequirements('test1!'))
+        assert.isFalse(signup.meetRequirements('test'))
```

Comment:
Can you add back the number and the symbol and remove some letters instead? The goal of this is to test the password being too short, and if we remove the symbols, we might see this check returning false because the password is missing a symbol or a number rather than because the password is too short.

Reply:
oh gotcha didn't think of that